src/msw/listctrl.cpp

```diff
@@ -334,6 +334,11 @@ void wxListCtrl::MSWSetExListStyles()
         // it seems better to enable it by default than disable
         LVS_EX_HEADERDRAGDROP
     );
+
+    // If using the default background style with double buffering enabled then switch to system
+    // Using wxBG_STYLE_ERASE while will cause flicker despite double buffering
```

Comment:
Sorry, I don't understand this. Was "while" supposed to be "style"?

In any case, changing `m_backgroundStyle` here seems really unexpected. Perhaps we could rather check for `IsDoubleBuffered()` when checking for it instead?